ios/Sources/SettingFeature/SettingScreen.swift

```diff
@@ -0,0 +1,56 @@
+import Styleguide
+import SwiftUI
+
+public struct SettingToggleModel: Hashable {
+    let title: String
+    var isOn: Bool
+}
+
+public struct SettingScreen: View {
+
+    @State private var items: [SettingToggleModel]
+
+    @Environment(\.presentationMode) var presentationMode
+
+    public init(isDarkModeOn: Bool, isLaunguageOn: Bool) {
+        let darkModeModel = SettingToggleModel(
+            title: L10n.SettingScreen.ListItem.darkMode,
+            isOn: isDarkModeOn
+        )
+        let languageModel = SettingToggleModel(
+            title: L10n.SettingScreen.ListItem.language,
+            isOn: isLaunguageOn
+        )
+        _items = State(initialValue: [darkModeModel, languageModel])
+    }
+
+    public var body: some View {
```

Comment:
Please fit background color and navigationTitle tintColor 🙏🏼 
I think it can change with `NavigationBar.appearance().**

If other view's NavigationBar color changed, please try to reset NavigationBar color on `deinit` .

Reply:
I do not clearly understand meaning of `If other view's NavigationBar color changed, please try to reset NavigationBar color on deinit .`. 😓

Reply:
I try fixing this by configuring `UINavigationBarAppearance` at `SettingScreen` iinitialization😊 
Does it clear your concern?

Reply:
> I do not clearly understand meaning of If other view's NavigationBar color changed, please try to reset NavigationBar color on deinit .. 😓

伝えづらいところなので日本語で行きます :pray:

SwiftUI で NavigationBar の色を変えるには `UINavigationBar.appearance()` を用いるしかないと思うんですが、
これは変えると他の箇所にも影響（色が変わってしまうなど）するのかなと思っていて、本当に影響するのかどうかは定かではないので、もし影響していたら `SettingScreen` で行った `UINavigationBar.appearance()` に対する変更を元に戻す変更を `deinit` 時に行うと他の箇所への影響が打ち消せるかなと思ってコメントした次第です！

他の箇所への影響等なければ何も対応することはないかと思います！

Reply:
> 一旦NavigationBarConfiguratorを作ってそこでonDisappearする際に前の色に戻す実装にしてみました。具体的な挙動が掴めていないので修正が必要であるか画面遷移が実装されるタイミングで確認してもらえると嬉しいです! 🙏

実際に手元で挙動に問題がないか確認するので、少しお待ちください🙏